Dockerfile

```
@@ -19,8 +21,7 @@ WORKDIR /app
 RUN npm install
```

```suggestion
RUN npm install-clean
```

(Suggestion, non blocking): we should only install dependencies from the lockfile. npm install does not consider the lockfile as readonly.  https://www.geeksforgeeks.org/difference-between-npm-i-and-npm-ci-in-node-js/. Also it is said to be faster 🐎

---

Nice, I did not know this one. Let me update